### PR TITLE
Added an option to run the script with windows system

### PR DESCRIPTION
**Pull request by Tal-D**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -1,3 +1,20 @@
 #!/bin/bash
 
-docker run --rm -it -v $(pwd):/app-dir/local -w /app-dir/local contacts-fixer sh
+# Choose running machine name
+unameOut="$(uname -s)"
+case "${unameOut}" in
```
Comment (georgettica) [inline run.sh:5]: No need for the unameOut 
```suggestion
 case "$(inane -s)" in
```

File: run.sh [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
+    MINGW*)     machine=MinGw;;
+    *)          machine="UNKNOWN:${unameOut}"
+esac
+
+# Mount repository path
+if [ $machine == "Linux" ]
+then
+	docker run --rm -it -v $(pwd):/app-dir/local -w /app-dir/local contacts-fixer sh
+else
+	windows_path=$(pwd | sed 's/^\///' | sed 's/\//\\/g' | sed 's/^./\0:/')
```
Comment (georgettica) [inline run.sh:18]: Is there a cleaner way to do that? It seems that there should be a cleaner way (without sed magic)

File: run.sh [hunk 3]
```diff
@@ -1,3 +1,20 @@
...
+	winpty docker run --rm -it -v $windows_path:/app-dir/local contacts-fixer sh
```
Comment (georgettica) [inline run.sh:19]: Why is winpty required?

**Review by georgettica — COMMENTED** [review]
Wrote some stuff, some are easier and some a re queries regarding why you chose to do things this way

**Comment by georgettica** [comment]
Also, congratulations on the contribution 🥳🎉

File: run.sh (hunk 3 above)
Comment (Tal-D) [reply run.sh:19]: Removed. Found a different solution - running with bash instead of git-bash.

File: run.sh (hunk 2 above)
Comment (Tal-D) [reply run.sh]: Started by reducing the number of calls to sed command.

Comment (Tal-D) [reply run.sh]: Fixed

**Review by georgettica — APPROVED** [review]

**Comment by georgettica** [comment]
Good job! And thanks

**Pull request merged**